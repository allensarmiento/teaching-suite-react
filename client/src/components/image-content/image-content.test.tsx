import { shallow } from 'enzyme';
import ImageContent from './image-content.component';

it('renders one image component', () => {
  expect(shallow(<ImageContent content={['//:0']} />)).toMatchSnapshot();
});

it('renders mulitple images', () => {
  expect(shallow(<ImageContent content={['//:0', '//:0']} />))
    .toMatchSnapshot();
});
