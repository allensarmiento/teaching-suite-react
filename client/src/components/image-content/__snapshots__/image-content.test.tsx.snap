// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders mulitple images 1`] = `
<Fragment>
  <div
    className="container"
    key="0"
  >
    <img
      alt=""
      className="image"
      src="//:0"
    />
  </div>
  <div
    className="container"
    key="1"
  >
    <img
      alt=""
      className="image"
      src="//:0"
    />
  </div>
</Fragment>
`;

exports[`renders one image component 1`] = `
<Fragment>
  <div
    className="container"
    key="0"
  >
    <img
      alt=""
      className="image"
      src="//:0"
    />
  </div>
</Fragment>
`;
